### PR TITLE
GNS3 2.2.25

### PR DESCRIPTION
**Pull request by tsndqst**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

#### Does it build and run successfully? 
(Please choose at least one native build and, if supported, at least one cross build. More are better.)
- [x] I built this PR locally for my native architecture, (x86_64-glibc)
- [ ] I built this PR locally for these architectures (if supported. mark crossbuilds):
  - [ ] aarch64-musl
  - [ ] armv7l
  - [ ] armv6l-musl

### CODE REVIEW

File: srcpkgs/gns3-server/template [hunk 1]
```diff
@@ -13,16 +13,9 @@ maintainer="Tim Sandquist <tim.sandquist@gmail.com>"
...
-post_patch() {
-	# relax requirements
-	vsed -e 's|aiofiles.*|aiofiles>=0.4.0|' \
-		-e 's|aiohttp=.*|aiohttp>=3.7.4|' \
-		-e 's|jsonschema.*|jsonschema>=2.6.0|' \
-		-e 's|py-cpuinfo.*|py-cpuinfo>=5.8.0|' -i requirements.txt
-}
 post_install() {
 	# remove tests directory polluting site-packages
 	rm -rf -- ${DESTDIR}/usr/lib/python*/site-packages/tests
```
Comment (paper42) [inline srcpkgs/gns3-server/template:21]: I would prefer `rm -r`, so it starts failing when it's not necessary anymore.

File: srcpkgs/python3-sentry/template [hunk 2]
```diff
@@ -12,7 +12,7 @@ maintainer="Karol Kosek <krkk@krkk.ct8.pl>"
 license="BSD-2-Clause"
 homepage="https://github.com/getsentry/sentry-python"
 distfiles="https://github.com/getsentry/sentry-python/archive/${version}.tar.gz"
-checksum=@514331143990028b4cecf9f8e4ff5dfe4c0dddd0dcb206b28b7fba22ada6cece
+checksum=45f0893c9b06df88ae6fff8ca7a2b33975b8e8b62aa3cae6f2b4704754f2c9fd
 
 do_check() {
 	cd build/lib
```
Comment (paper42) [inline srcpkgs/python3-sentry/template:18]: I think we could run tests with pytest, no?

File: srcpkgs/gns3-server/patches/requirements.patch [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+--- a/requirements.txt
++++ b/requirements.txt
+@@ -1,10 +1,10 @@
+ jsonschema==3.2.0
+-aiohttp==3.7.4.post0
++aiohttp>=3.7.4
+ aiohttp-cors==0.7.0
+ aiofiles==0.7.0
+-Jinja2==3.0.1
++Jinja2<3.0.0
```
Comment (paper42) [inline srcpkgs/gns3-server/patches/requirements.patch:12]: it should be possible to do something like Jinja2>=2.0.0<4.0.0

**Review by paper42 — COMMENTED** [review]
Next time you can force push to a PR instead of opening a new one.

**Comment by tsndqst** [comment]
> Next time you can force push to a PR instead of opening a new one.

True.  I made a new PR because I made a new branch.  The old branch had the 2.2.23 version number in it so I wanted to avoid confusion by creating a new branch with the correct name.

**Comment by tsndqst** [comment]
I missed something in gns3-gui.  Working on a fix now.

**Comment by tsndqst** [comment]
@paper42 can you take another look?  I did a bit of refactoring to get tests working instead of skipping them altogether.  It involved adding a couple packages but they were pretty straight forward.

I also removed python3-sentry package since it was only used for crash reports and apparently isn't included in the upstream packages.

**Comment by paper42** [comment]
python3-sentry needs to be added to removed-packages like in #32495

GitHub doesn't detect you as the author of the commits, could you add the email you are using in the commits to your GitHub profile?

**Comment by tsndqst** [comment]
Thanks @paper42 
I'll make a separate PR for removal of python3-sentry once this PR is merged.
I updated my email address in GitHub as well.

**Pull request merged**